# genomic_clines_input
R scripts used by Blackburn et al. 2017 to translate SNP data into input files for BGC (Gompert and Buerkle 2011) genomic clines analysis.

Notes:

The script assumes zero genotype error (see Gompert and Buerkle 2012).

Example files are provided for a fake admixed population (n=100 individuals) and two fake parental populations (n=30 individuals each), each scored for 10 SNPs.


References:

Blackburn G. S., Bryan M.T. Brunet, Kevin Muirhead, Michel Cusson, Catherine Béliveau, Roger C. Levesque, Lisa M. Lumley, and Felix A.H. Sperling. Distinct sources of gene flow produce contrasting population genetic dynamics at different range boundaries of a Choristoneura budworm. XXXX

Gompert, Z., and C. A. Buerkle. 2011. Bayesian estimation of genomic clines. Mol. Ecol. 20:2111–2127

Gompert, Z., and C. A. Buerkle. 2012. bgc: Software for Bayesian estimation of genomic clines. Mol. Ecol. Resour. 12:1168–1176
